### ISSUE TITLE
Wheel not working on Windows

### ISSUE THREAD

**Issue by pistou** [jl-/react-slide-deck#6]
It appears that scrolling is not working.
When I set `wheel: true`, scrolling works **only for the first slide** and **only the very first time**.

If you try to scroll again once on the second slide, nothing happens. Even if you come back _manually_ to the first slide, you can't scroll anymore.

[Your example](http://output.jsbin.com/hexada) has the same behaviour.

For the record, I tried on Chrome (Version 47.0.2526.111 m) only.
I just updated Chrome to v48.0 - still not working.

**Comment by jl-** [comment]
That's weird, I just open the example page, clear chrome(48.0) browser cache, and it works as expected, woks fine on firefox as well expect for some jank animation. However, there's an similar issue on safari as wheel event behaves differently. Hopefully it will be fixed for Safari on next release, but for chrome, without reproducing the bug, I have no idea what might be the source of the problem. I'm using mac, are you on windows? platform difference leads to chrome's different behaviour, I could only guess for now.

**Comment by pistou** [comment]
Yes I'm on Windows. Doesn't work on both Windows 7 and Windows 10 :disappointed: 

I'll try to tweek the sources and debugg it on my own as soon as I get a little time.

**Comment by jl-** [comment]
That would be awesome !! Here are some hints, hope it would help you out!
- [handleWheel](https://github.com/jl-/react-slide-deck/blob/master/src/deck.js#L150), 
- [a discuss](https://github.com/jl-/react-slide-deck/pull/3)

Thanks again and good luck!

**Comment by aga5tya** [comment]
@jl- This issue still exists on windows with mouse. But can't be replicated on osx, with both mouse and trackpad.

The issue lies with the below line, post commenting this, everything works fine. Leaving this here for anyone who still has the issue on windows and mouse wheel.
https://github.com/jl-/react-slide-deck/blob/master/src/deck.js#L184
